web/themes/custom/cj_material/sass/styles.scss

```diff
@@ -1375,11 +1376,282 @@ article footer {
   font-style: italic;
 }
 
+/* BP - Job Post Page */
+
 .form-item label {
-  font-size: 14px;
+  font-size: 20px;
   font-weight: bold;
+  font-family: Montserrat;
+  line-height: 2em;
 }
 
+.form-item input[type=text], .form-item input[type=email], .form-item input[type=password], .form-item input[type=search], .form-item input[type=tel], .form-item textarea, .form-item select {
+    display: block;
+    width: 100%;
+    max-width: 1200px;
+    border: 1px solid rgba(0, 0, 0, 0.12);
+    padding: 8px 0;
+}
+
+input#edit-label-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+textarea#edit-description-summary-0-value {
+    background-color: #f6f4f4;
+    text-indent: 10px;
+}
+
+form#job-role-post-form {
+    padding-left: 20px;
+    padding-right: 20px;
+}
+
+/* BP Post a Job 21/1/2020 */
+
+summary {
+    border: 2px solid;
+    border-color: #ff8f48;
+}
+
+.divider-top:before {
+    background: #FF904B;
+    content: "";
+    height: 6px;
+    position: absolute;
+    left: -40px;
+    top: 0px;
+    width: 1200px;
+}
+
+.form-item input[type=text]:focus, .form-item input[type=text].error:focus, .form-item input[type=email]:focus, .form-item input[type=email].error:focus, .form-item input[type=password]:focus, .form-item input[type=password].error:focus, .form-item input[type=search]:focus, .form-item input[type=search].error:focus, .form-item input[type=tel]:focus, .form-item input[type=tel].error:focus, .form-item textarea:focus, .form-item textarea.error:focus, .form-item select:focus, .form-item select.error:focus {
```

Comment:
Not sure why on of these is only applying with .error. Good spot though! When the fields have failed validation we should border them in red.

Use nested declarations as above.